docs/tokenomics/nns/proposal-requirements.md

```diff
@@ -0,0 +1,206 @@
+
+The NNS assigns a unique identity to each proposal that it receives.
+
+### Topics
+
+Each type of proposal belongs to a specific “proposal topic,” such as “#NodeAdmin” or “#NetworkEconomics,” which determines details about how it will be processed. To prevent users (neurons) from spamming the NNS with proposals, a fee is levied on the neuron that submitted a proposal if it is rejected.
+Proposals are adopted or rejected based on voting activity by “neurons” that network participants have created.
+
+For example, the NNS may require voters to have a greater degree of agreement or to try to process proposals faster, for some topics. Also, neurons follow other neurons on a per-topic basis. Initial topics include:
+
+* **#NeuronManagement:** a special topic using which a neuron can be managed by the followees for this topic (in this case, there is no fallback to default). Votes on this topic are not included in the voting history of the neuron. For proposals on this topic, only the neuron’s followers on the topic that the proposals pertain to are allowed to vote. Because the set of eligible voters for proposals on this topic is restricted, proposals on this topic have a shorter than normal voting period.
+* **#ExchangeRate:** all proposals provide information in “real-time” about the market value of ICP, as measured by an International Monetary Fund (IMF) Special Drawing Right (SDR), which allows the NNS to convert ICP to cycles (which power computation) at a rate that keeps their real-world cost constant. Because proposals on this topic are very frequent, they have a shorter voting period, and votes on this topic are not included in the voting history of the neuron.
+* **#NetworkEconomics:** proposals that administer network economics — for example, determining what rewards should be paid to node operators.
+* **#Governance:** all proposals that administer governance — for example, motions and the configuration of certain parameters.
+* **#NodeAdmin:** all proposals that administer node machines somehow, including but not limited to upgrading or configuring the OS, upgrading or configuring the virtual machine framework, and upgrading or configuring the node replica software.
+* **#ParticipantManagement:** all proposals that administer network participants — for example, granting and revoking DCIDs (data center identities) or NPIDs (node provider identities).
+* **#SubnetManagement:** all proposals that administer network subnets — for example, creating new subnets, adding and removing subnet nodes, and splitting subnets.
+* **#NetworkCanisterManagement:** installing and upgrading “system” [[NNS Canisters|canisters]] that belong to the network — for example, upgrading the NNS.
+* **#KYC:** proposals that update KYC information for regulatory purposes — for example, during the initial Genesis distribution of ICP in the form of neurons.
+* **#NodeProviderRewards:** topic for proposals to reward node providers.
```

Comment:
not sure if this is still used. another candidate to remove.
Maybe instead one coud mention SNS proposals that are used more frequently?